src/Scripting/CSharpTest/ObjectFormatterTests.cs

```diff
@@ -950,13 +953,11 @@ public void StackTrace_Dynamic()
             }
             catch (Exception e)
             {
-                const string filePath = @"z:\Fixture.cs";
-
                 var expected =
 $@"'object' does not contain a definition for 'x'
   + System.Dynamic.UpdateDelegates.UpdateAndExecuteVoid1<T0>(System.Runtime.CompilerServices.CallSite, T0)
-  + Microsoft.CodeAnalysis.CSharp.Scripting.Hosting.UnitTests.ObjectFormatterTests.Fixture2.MethodDynamic(){string.Format(ScriptingResources.AtFileLine, filePath, 10123)}
-  + Microsoft.CodeAnalysis.CSharp.Scripting.Hosting.UnitTests.ObjectFormatterTests.StackTrace_Dynamic(){string.Format(ScriptingResources.AtFileLine, filePath, 10132)}
+  + Microsoft.CodeAnalysis.CSharp.Scripting.Hosting.UnitTests.ObjectFormatterTests.Fixture2.MethodDynamic(){(s_supportsLineNumbers ? string.Format(ScriptingResources.AtFileLine, StackTraceSourceFile, 10124) : "")}
```

Comment:
💭 The result is different for this test on **ubuntu_16_debug_prtest** (produces line 10123) and **windows_coreclr_test_prtest** (produces line 10124).